config/config.go

```diff
@@ -166,26 +168,46 @@ func (c *CombinerConfig) initComponent() (fiber.Component, error) {
 // ProxyConfig is used to parse the configuration for a Proxy
 type ProxyConfig struct {
 	ComponentConfig
-	Endpoint string   `json:"endpoint" required:"true"`
-	Timeout  Duration `json:"timeout"`
+	Endpoint          string   `json:"endpoint" required:"true"`
+	Timeout           Duration `json:"timeout"`
+	Protocol          string   `json:"protocol"`
+	Service           string   `json:"service"`
+	Method            string   `json:"method"`
```

Comment:
nit: Having `ResponseProtoName` is redundant since we are using server reflection. Using the `Method`, you can actually figure out the request and response proto from the service descriptor

Reply:
<img width="678" alt="Screenshot 2022-08-30 at 4 13 08 PM" src="https://user-images.githubusercontent.com/4023015/187385645-9860f6b2-440d-43e5-8106-5e15f24c3c29.png">

Reply:
thats a fair point, let me investigate further if we can eliminate this